### PR TITLE
Register type-id for asAscCdl utility shader

### PR DESCRIPTION
**Pull request by Mango-3**

### CODE REVIEW

File: src/appleseedmaya/typeids.h [hunk 1]
```diff
@@ -109,6 +109,7 @@ enum AppleseedMayaTypeIds
     AsSbsPbrMaterialNodeTypeID      = 0x00127a04,   // 1210884
     AsManifold2DNodeTypeID          = 0x00127a05,   // 1210885
     AsManifold3DNodeTypeID          = 0x00127a06,   // 1210886
+    AsAscCdlNodeTypeId              = 0x00127a07,   // 1210887
```
Comment (dictoon) [inline src/appleseedmaya/typeids.h:112]: While you're modifying this file, could you replace the `ID` suffixes by `Id` to make naming consistent?  Indentation also needs to be fixed.

Comment (Mango-3) [reply src/appleseedmaya/typeids.h:112]: Id suffixes corrected. But I can't see what is the issue with the indentation

Comment (dictoon) [reply src/appleseedmaya/typeids.h:112]: Thanks!

The indentation "problem" was simply a glitch of GitHub's UI (maybe a small regression on their side, I haven't seen this before):

image

Comment (dictoon) [reply src/appleseedmaya/typeids.h:112]: Don't the renamings affect other code?

Comment (dictoon) [reply src/appleseedmaya/typeids.h:112]: Apparently not... Merging!

**Pull request merged**